.dockerignore

```diff
@@ -1,4 +1,3 @@
-.git
```

Comment:
I see docker build has the same issue as nix build where the .git directory is not provided. I don't think we should copy the .git folder as it will invalidate the cache.

Instead, I think we should fallback to an env variable using th-env to be set with `MONOCLE_COMMIT=$(git show HEAD --format="format:%H" -q)`